src/sponge/sponge.rs

```diff
@@ -0,0 +1,203 @@
+//! The `pad` module implements the Sponge's padding algorithm
```

Comment:
It's a bit unclear to me this comment.

Reply:
Not targetting to have a good documentation atm.

I just put this so I don't get errors due to the flag #[deny(missing_docs)].
I was leaving docs for the end once #11 gets merged.